lxc config trust dumps stack

Exploring the cli, I ran lxc config trust and it dumped stack.

Reproduce:

1. get lxd, deps and build it with make, then run
~/go/bin/lxd --debug --tcp 127.0.0.1:8443

2. ~/go/bin/lxc config trust

Error Output:
% ~/go/bin/lxc config trust 
panic: runtime error: index out of range

goroutine 16 [running]:
runtime.panic(0x736880, 0xb2db9c)
	/usr/lib/go/src/pkg/runtime/panic.c:279 +0xf5
main.(*configCmd).run(0xc208000c10, 0xc20804c340, 0xc208000c80, 0x1, 0x1, 0x0, 0x0)
	/home/rharper/go/src/github.com/lxc/lxd/lxc/config.go:52 +0xf88
main.run(0x0, 0x0)
	/home/rharper/go/src/github.com/lxc/lxd/lxc/main.go:63 +0x74e
main.main()
	/home/rharper/go/src/github.com/lxc/lxd/lxc/main.go:15 +0x26

goroutine 19 [finalizer wait]:
runtime.park(0x420990, 0xb43230, 0xb2fe49)
	/usr/lib/go/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0xb43230, 0xb2fe49)
	/usr/lib/go/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
	/usr/lib/go/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
	/usr/lib/go/src/pkg/runtime/proc.c:1445

goroutine 17 [syscall]:
runtime.goexit()
	/usr/lib/go/src/pkg/runtime/proc.c:1445

--- a/lxc/config.go
+++ b/lxc/config.go
@@ -32,6 +32,10 @@ func (c *configCmd) run(config *lxd.Config, args []string) error {
 	switch args[0] {
 
 	case "set":
+		if len(args) < 2 {
+			return errArgs
+		}
+
 		action := args[1]
 		if action == "password" {
 			if len(args) != 3 {
@@ -50,6 +54,10 @@ func (c *configCmd) run(config *lxd.Config, args []string) error {
 
 		return fmt.Errorf(gettext.Gettext("Only 'password' can be set currently"))
 	case "trust":
+		if len(args) < 2 {
+			return errArgs
+		}
+
 		switch args[1] {
 		case "list":
 			var remote string

